notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Would it be possible to add links to the headers on the page for the following actions as well:</p><ul><li>Get a template and prepare it for use</li><li>Subtract the continuum from the observed spectrum</li><li>Run the cross correlation function</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done!